addon-test-support/ember-qunit/adapter.js

```diff
@@ -1,5 +1,6 @@
 import Ember from 'ember';
 import QUnit from 'qunit';
+import hasEmberVersion from '@ember/test-helpers/has-ember-version';
```

Comment:
is that helper public API?

Reply:
Its effectively public, its basically in the same state as `wait` was before. Lots of folks use it, but we haven't officially said its "public". 

I'll open a PR to export it from the entry point...